@navikt/core/react/src/util/hooks/descendants/descendant.ts

```diff
@@ -9,6 +9,11 @@ export type DescendantOptions<T = object> = T & {
    * but omitted from enabled nodes map
    */
   disabled?: boolean;
+  handleVirtualOnFocus?: (
+    node_to_focus: HTMLElement,
+    node_to_blur?: HTMLElement,
+  ) => void;
+  handlePick?: () => void;
```

Comment:
These should not be typed here, but rather when creating the descendant context. (Added a comment with howto above 🤞 )